test_client.ts

```diff
@@ -0,0 +1,4 @@
+import { SocketClient } from "./mod.ts";
```

Comment:
is this just a test app? if so, consider placing it in an `/example_app` dir. same goes for the `test_server.ts` file below